snooty/util.py

```diff
@@ -83,6 +83,14 @@ def is_relative_to(a: Path, b: Path) -> bool:
         return False
 
 
+def path_given_relative_url(relative_url: str, project_root: Path) -> str:
+    """Returns absolute path to txt file specified by relative URL.
+    URLs can be of the form /foo, foo, /foo/"""
+    relative_url = relative_url[1:] if relative_url.startswith("/") else relative_url
+    relative_url = relative_url[:-1] if relative_url.endswith("/") else relative_url
+    return f"{project_root}/{relative_url}.txt"
```

Comment:
We could probably make use of the `reroot_path` util function to handle some of this logic. I think `pathlib` can handle some of the logic with the forward slashes as well